backend/Controllers/SightingController.cs

```diff
@@ -2,17 +2,27 @@
 using WhaleSpotting.Services;
 using WhaleSpotting.Models.Response;
 using WhaleSpotting.Models.Database;
+using Microsoft.EntityFrameworkCore;
 
 namespace WhaleSpotting.Controllers {
     [ApiController]
     [Route("/sightings")]
     public class SightingController : ControllerBase {
         private readonly ISightingService _sightings;
 
-        public SightingController(ISightingService sightings)
-        {
+        public SightingController
+        (
+            ISightingService sightings
+        ) {
             _sightings = sightings;
         }
+        
+        [HttpGet("/approved")] 
```

Comment:
I think approved sightings should be the default, therefore a GET request simply to `/sightings` should return all _approved_ sightings:
```suggestion
        [HttpGet] 
```